Sources/TuistAcceptanceTesting/TuistAcceptanceFixtures.swift

```diff
@@ -131,6 +132,8 @@ public enum TuistAcceptanceFixtures {
             return "ios_app_with_custom_scheme"
         case .iosAppWithExtensions:
             return "ios_app_with_extensions"
+        case .iosAppWithExtensionHavingUnitTest:
+            return "ios_app_with_extension_having_unit_test"
```

Comment:
```suggestion
            return "ios_app_with_extension_and_tests"
```

nit regarding the fixture name

Reply:
fixed